modules/claims_api/app/controllers/claims_api/base_disability_compensation_controller.rb

```diff
@@ -16,12 +16,17 @@ class BaseDisabilityCompensationController < ClaimsApi::BaseFormController
 
     def upload_form_526
       pending_claim = ClaimsApi::AutoEstablishedClaim.pending?(params[:id])
-      pending_claim.set_file_data!(documents.first, params[:doc_type])
-      pending_claim.save!
 
-      ClaimsApi::ClaimUploader.perform_async(pending_claim.id)
+      if pending_claim
```

Comment:
One last thing I realized that should have been here but is missing is we need to check for if 

form_attributes['autoCestPDFGenerationDisabled'] == true

This is to ensure that no one ever uploads a 526 claim if one is being already generated by EVSS, was in the original spec for this end point but we must have missed it somehow. The error should probably say something like Claim not found or upload disabled due to PDF generation etc.